doc/classes/PopupMenu.xml

```diff
@@ -519,10 +551,13 @@
 			If [code]true[/code], allows to navigate [PopupMenu] with letter keys. Default value: [code]false[/code].
 		</member>
 		<member name="hide_on_checkable_item_selection" type="bool" setter="set_hide_on_checkable_item_selection" getter="is_hide_on_checkable_item_selection">
+			Determines whether the [PopupMenu] will be hidden when a checkbox or a radio button is selected or not.
```

Comment:
Use the present tense whenever possible. This is also in the writing guidelines.
Also, same as above, you can use more precise verbs than `determine`: `Hides ...` in this case

Reply:
So you propose `Hides or shows ....` over `determines?`

Reply:
`Hides` over `Determines whether`